app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,221 @@
+/* Global Styles */
+
+html,
+body {
+    height: 100%;
+}
+
+body {
+    padding-top: 42px; /* Required padding for .navbar-fixed-top. Remove if using .navbar-static-top. Change if height of navigation changes. */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: 400;
+}
+
+.navbar {
+  font-weight: 400;
+}
+
+.page-header {
+  border-bottom: none;
+  color: white;
+  font-weight: 500;
+  float: right;
+}
+
+.featured {
+  color: black;
+  float: none;
+}
+
+.main_image {
+  background: url("dew-drops-carnivorous-plant.jpg") no-repeat center center;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  height: 362px;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  background: no-repeat center center;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, height